relay-server/src/actors/events.rs

```diff
@@ -674,6 +690,19 @@ impl EventProcessor {
             // id was ingested, this will already be the case. Otherwise, this will insert a new
             // event id. To be defensive, we always overwrite to ensure consistency.
             event.id = Annotated::new(event_id);
+
+            // TODO: Temporary workaround before processing. Experimental JavaScript SDKs relied on
+            // a buggy clock drift correction that assumes the event timestamp is the sent_at date.
+            // This should be removed as soon as JS has switched to Envelope-only ingestion.
+            let client = envelope.meta().client().unwrap_or_default();
+            if envelope.sent_at().is_none()
+                && event_type == Some(EventType::Transaction)
+                && client.starts_with("sentry-javascript/")
```

Comment:
From JS:
`header.push(`sentry_client=${clientName}/${clientVersion}`);`

`clientName = sentry.javascript.browser`
```suggestion
                && client.starts_with("sentry.javascript")
```